chassis_windows.go

```diff
@@ -0,0 +1,40 @@
+// Use and distribution licensed under the Apache license version 2.
+//
+// See the COPYING file in the root project directory for full text.
+//
+
+package ghw
+
+import (
+	"github.com/StackExchange/wmi"
+)
+
+type CIM_Chassis struct {
```

Comment:
Same comment as above about using an unexported struct name and a const string for the WQL query.